<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SectorAndTotalNatDiagram</class>
 <widget class="QWidget" name="SectorAndTotalNatDiagram">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>201</width>
    <height>431</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>201</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>201</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="lineEditTotalNatDoc">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Archivo de la CC nacional:</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxDayType">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>141</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Días de los sectores</string>
   </property>
   <widget class="QRadioButton" name="radioButtonWorkdays">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Lunes a viernes</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonSaturdays">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Sábados</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonSundays">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Domingos</string>
    </property>
   </widget>
  </widget>
  <widget class="QComboBox" name="comboBoxSelectSector">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Seleccione sector</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxShowSubCats">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Mostrar subcategorías</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
